test/testLauncher/SuitestLauncher.test.js

```diff
@@ -14,28 +14,28 @@ const envVars = require('../../lib/constants/enviroment');
 const logger = require('../../lib/utils/logger');
 
 describe('SuitestLauncher', () => {
-	before(async() => {
+	before(async () => {
```

Comment:
Why these changes, we don't have this enforce din eslint?

Reply:
my editor did this automatically. 
If you prefer i can revert this. 
It would be great if we have all IDE settings in this project somehow synchronized.

Reply:
The rule is actually present in eslint: `"space-before-function-paren": ["warn", "never"]`, space should not be there

Reply:
right. But i have applied autoformatting (Ctrl+Alt+L) on idea and it did this. I think we should check the .editorconfig or something. I will revert this file.